tests/ui/test/features/display-no-results.feature

```diff
@@ -0,0 +1,8 @@
+Feature: As a user I want to be able to see no results screen displayed
```

Comment:
This feature won't just be about seeing the no results screen. Is it the start of the Add organisation path. Think that it should sit in the existing Organisations so that it matches up with the steps.
We will need to move out all the Add Organisation functionality at a later point

Reply:
OK. Happy to move them